gradle.properties

```diff
@@ -33,12 +33,12 @@ POM_DEVELOPER_ID=ashyonline
 POM_DEVELOPER_NAME=Ayelen Chavez
 POM_DEVELOPER_EMAIL=ashi@creativa77.com.ar
 
-NEXUS_USERNAME=xxx
-NEXUS_PASSWORD=yyy
+NEXUS_USERNAME=ashyonline
+NEXUS_PASSWORD=jO7wq4Ai3
 
 SNAPSHOT_REPOSITORY_URL=https://oss.sonatype.org/content/repositories/snapshots
 RELEASE_REPOSITORY_URL=https://oss.sonatype.org/service/local/staging/deploy/maven2
 
-signing.keyId=xxxx
-signing.password=yyyy
-signing.secretKeyRingFile=/home/<user>/.gnupg/secring.gpg
\ No newline at end of file
+signing.keyId=BD9F6336
+signing.password=eku77men
+signing.secretKeyRingFile=/home/ashi/.gnupg/secring.gpg
```

Comment:
This is committing a change with a path pointing to your computer, is this correct? Shouldn't the key be in the repo?